src/t2iapi/combined/service.proto

```diff
@@ -51,4 +51,15 @@ service CombinedService {
   rpc SetAlertActivationAndSetOperatingMode (
   t2iapi.combined.SetAlertActivationAndSetOperatingModeRequest)
       returns (BasicResponse);
+
+  /*
+  Request to set the non-exclusive mode of operation represented by an enum metric with the given handle to the
```

Comment:
"Set the ..." instead of "Request to set the ..."

Reply:
After discussion, the wording is not my preferred choice but it's ok for me. We can leave it as it is.